connectivity/check/features.go

```diff
@@ -212,7 +212,7 @@ func (ct *ConnectivityTest) extractFeaturesFromConfigMap(ctx context.Context, cl
 	}
 
 	if versioncheck.MustCompile("<1.14.0")(ct.CiliumVersion) {
-		mode = "disabled"
+		mode = "vxlan"
```

Comment:
Mind extending the commit message explaining what problem this commit tries to solve?